content/blog/2024/openscapes-sbg-workshop/index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Openscapes host a Surface Biology and Geology workshop
+date: "2024-05-29"
+banner:
+  image: "featured.png"
+authors: ["Yuvi Panda", "Jenny Wong"]
+tags: [geoscience, education]
+categories: [impact, updates, partnerships]
+featured: false
+draft: false
+---
+
+![Cover slide from the SBG Workshop](featured.png "Surface Biology and Geology: VITALS Workshop")
+
+[Openscapes](https://nasa-openscapes.github.io/) is a value-based initiative that advocates for kinder, better science produced with open source community and program building co-developed by Julia Lowndes and Erin Robinson. 
+
+The initiative recently supported the [Surface Biology and Geology: VITALS Workshop](https://nasa.github.io/VITALS/) hosted by NASA [Land Processes Distributed Activate Archive Center (LP DAAC)](https://lpdaac.usgs.gov/) and NASA [Jet Propulsion Laboratory (JPL)](https://www.jpl.nasa.gov/).
+
+Hands-on exercises teaching learners how to manipluate data collected from the [ECOSTRESS](https://ecostress.jpl.nasa.gov/) and [EMIT](https://earth.jpl.nasa.gov/emit/) instruments onboard the International Space Station were delivered using the 2i2c OpenScapes Hub. [Jupyter notebooks](https://nasa.github.io/VITALS/python/01_Finding_Concurrent_Data.html) were used to demonstrate how open source tools together with cloud data and compute resources could effectively analyse the the Canopy Water Content and the Land Surface Temperature over the [Jack and Laura Dangermond Preserve](https://www.dangermondpreserve.org/), Santa Barbara, CA.
+
+![Plot of the Canopy Water Content over the Jack and Laura Dangermond Preserve, Santa Barbara, CA.](canopy-water-content.png "Plot of the Canopy Water Content over the Jack and Laura Dangermond Preserve, Santa Barbara, CA from a [VITALS Workshop Jupyter notebook](https://nasa.github.io/VITALS/python/03_EMIT_CWC_from_Reflectance.html).")
+
+This event was attended by around 250 participants. To facilitate a smooth hub log-in process that didn't rely on GitHub authentication, we set up a shared password that workshop organizers can hand out to learners for access. This bypassed the manual labour of sending out GitHub invitations and drastically reduces the amount of preparatory work for events at this scale, so that organizers could focus on the essential complexity of teaching data analysis rather than the accidental complexity of managing hub authorization.
```

Comment:
instead of 'smooth' login in process... maybe something more like instantaneous.   Even though instantaneous login was probably the experience participants expected (because its 2024), prior to the shared login in option; In advance of workshops:

1. workshop leaders retrieve participant GitHub handle (with potential added work of participant making an account) 
2. workshop leaders add participant added to Openscapes space 
3. which triggers participant receiving a sketch email from GitHub with link that expires in 7 days to be added to Openscapes Team Space
4. workshop leaders check to see how many participants need to be added to team space
5. workshop leaders still have to take time out of "hands on learning" to get everyone started in the same computation space

Reply:
I think there is also a lot of _value_ in not adding to high cognitive load by having participant frustration..... that is just really fantastic :)

Reply:
I think this is great to contrast. Could frame this as "for some longer workshops with smaller numbers of participants where a workflow with GitHub is part of the lesson, the following approach can work well. However, we were interested in a more instantaneous process for short workshops with large numbers of participants where GitHub is not part of the lesson.

Reply:
I've taken your comments and expanded on this point in https://github.com/2i2c-org/2i2c-org.github.io/pull/274/commits/1db469b8c8a4a3343c0634babcbf3b6e758d5696 😄